example/spec/parallel/ingestGranule/IngestUMMGSuccessSpec.js

```diff
@@ -345,7 +346,7 @@ describe('The S3 Ingest Granules workflow configured to ingest UMM-G', () => {
       const viewRelatedInfoResource = onlineResources.filter((resource) => resource.Type === 'VIEW RELATED INFORMATION');
       const s3CredsUrl = resolve(process.env.DISTRIBUTION_ENDPOINT, 's3credentials');
 
-      const ExpectedResources = ['GET DATA', 'GET DATA', 'GET RELATED VISUALIZATION',
+      const ExpectedResources = ['GET DATA', 'GET DATA', 'GET DATA', 'GET RELATED VISUALIZATION',
```

Comment:
I assume we don't feel the need to ensure that the actual OpenDAP URL appears in resource URLs for UMM-G? Would there be value in making sure given the differences in handling for ECHO10 and UMM-G?

Reply:
I added it